### PR TITLE
feat: add transfer page

### LINKED ISSUE #24: Implement transfer Screen
Hello contributors/team,

Your focus will be on creating a visually appealing Transfer Screen.

Text, Icons, images input  and button to match the [design](https://www.figma.com/file/b9ysoeaLAFcjy2Dlrbu5Ya/Web-Map-Demo?type=design&node-id=429-2925&mode=design&t=hUHsIkIUmesFdD1u-0).
UI Design:
Screen Shot 2023-07-29 at 7 21 50 PM

Feel free to reach out if you have any questions or need further clarification.

### PR DESCRIPTION
**Pull request by dagmawig**
fixes #24

### CODE REVIEW

**Comment by dagmawig** [comment]
@pierrelstan  please review.

**Comment by dadiorchen** [comment]
@dagmawig is it easy for you to add screenshot for this?

**Comment by pierrelstan** [comment]
@dagmawig  please resolve the conflicts

File: src/screens/TransferScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+import { Svg, Path, SvgProps } from "react-native-svg";
+
+const TransferScreen = () => {
+  const bannerURI = Asset.fromModule(
+    require("../../assets/transfer-oval.png"),
+  ).uri;
+  const profileURI = Asset.fromModule(
+    require("../../assets/transfer-profile.png"),
+  ).uri;
+  const ArrowLeft = (props: SvgProps) => (
```
Comment (pierrelstan) [inline src/screens/TransferScreen.tsx:22]: Add the svgs to the Icons component.

File: src/screens/TransferScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      <Path
+        fill="#9597A2"
+        d="m0 12.5 2.211-2.21 8.726 8.726V0h3.126v19.016l8.726-8.727L25 12.5 12.5 25 0 12.5Z"
+      />
+    </Svg>
+  );
+
+  return (
+    <SafeAreaView style={{ ...style.container }}>
+      <View style={style.header}>
```
Comment (pierrelstan) [inline src/screens/TransferScreen.tsx:52]: Use the header that from the react navigation

File: src/screens/TransferScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+            </View>
+            <View>
+              <Text
+                style={{ fontSize: 11, fontWeight: "bold", color: "#61697D" }}>
+                Created at Jul 9, 2022h
+              </Text>
+            </View>
+          </View>
+        </View>
+        <View>{ArrowDown(null)}</View>
```
Comment (pierrelstan) [inline src/screens/TransferScreen.tsx:94]: Add the svgs icons as a component.  

for example: 
`<Header />`

File: src/screens/TransferScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+      </View>
+      <View style={{ flexDirection: "row", height: 60 }}>
+        <TextInput
+          style={{
```
Comment (pierrelstan) [inline src/screens/TransferScreen.tsx:98]: Reduce the inline css.

**Review by pierrelstan — CHANGES_REQUESTED** [review]
Replace  the textInputcomponent to  customInput component and customize it (customInput) to make it dynamic.
Do the same for the CustomButton component.

**Comment by dagmawig** [comment]
> @dagmawig is it easy for you to add screenshot for this?

image

**Review by pierrelstan — APPROVED** [review]

**Comment by pierrelstan** [comment]
great Work @dagmawig

**Pull request merged**